codewiki/src/be/utils.py

```
@@ -59,149 +62,155 @@ async def validate_mermaid_diagrams(md_file_path: str, relative_path: str) -> st
             error_pattern = r"Error:(.*?)(?=Stack Trace:|$)"
             match = re.search(error_pattern, error_str, re.DOTALL)
-            
+
             if match:
                 core_error = match.group(0).strip()
                 core_error = core_error
             else:
-                logger.error(f"No match found for error pattern, fallback to mermaid-py\n{error_str}")
+                logger.error(
+                    f"No match found for error pattern, fallback to mermaid-py\n{error_str}"
+                )
```

Ruff (0.14.10)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)